api/src/app/modules/analytics-dashboard/analytics-dashboard.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from '@nestjs/common';
+import {
+  ProjectCountByDistrictResponse,
+  ProjectCountByForestClientResponse,
+} from '@api-modules/project/project.dto';
+import { ProjectService } from '@api-modules/project/project.service';
+import {
+  PublicCommentCountByDistrictResponse,
+  PublicCommentCountByCategoryResponse,
+  PublicCommentCountByProjectResponse,
+} from '@api-modules/public-comment/public-comment.dto';
+import { PublicCommentService } from '@api-modules/public-comment/public-comment.service';
+
+@Injectable()
+export class AnalyticsDashboardService {
```

Comment:
This class seems a bit pointless if it just does straight delegation to other services. Why not just use those other services within the dashboard controller then?

Reply:
I removed it.